content/applications/sales/crm/pipeline/merge_similar.rst

```diff
@@ -0,0 +1,122 @@
+:guilabel:`Similar Leads` smart button. Doing so opens a Kanban view that only displays similar
+leads/opportunities. Click on a card to view the details for the lead/opportunity, and confirm if
+they should be merged.
+
+Merging similar leads and opportunities
+=======================================
+
+.. important::
+   When merging, Odoo gives priority to whichever lead/opportunity was created in the system first,
+   merging the information into the first created lead/opportunity. However, if a lead and an
+   opportunity are being merged, the resulting record is referred to as an opportunity, regardless
+   of which record was created first.
+
+After confirming that the leads/opportunities should be merged, return to the Kanban view using the
+breadcrumb link, or by clicking the :guilabel:`Similar Leads` smart button. Click the
+:icon:`oi-view-list` :guilabel:`(list)` icon to change to list view.
+
+Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
+:icon:`fa-cog` :guilabel:`Action` icon at the top of the page, to reveal a drop-down menu. From
+that drop-down menu, select the :guilabel:`Merge` option to merge the selected opportunities or
+leads.
+
+When :guilabel:`Merge` is selected from the :icon:`fa-cog` :guilabel:`Action` drop-down menu, a
+:guilabel:`Merge` pop-up modal appears. In that pop-up modal, under the :guilabel:`Assign
+opportunities to` heading, optionally select a :guilabel:`Salesperson` and :guilabel:`Sales Team`
+from the appropriate drop-down menus.
+
+Below those fields, the leads/opportunities to merge are listed, along with their related
+information. To add more, click the :guilabel:`Add a line button`, check the box next to the desired
+leads/opportunities, then click the :guilabel:`Select` button. The additional selections now appear
+in the list. To merge those selected leads/opportunities, click :guilabel:`Merge`.
+
+Once all leads/opportunities are selected, click the :guilabel:`Merge` button. The pop-up window
+disappears, and the detail card for the merged opportunity/lead appears.
+
+.. image:: merge_similar/select-merge.png
+   :align: center
+   :alt: List of similar leads and opportunities selected for merge in the CRM app.
+
+.. danger::
+   Merging is an irreversible action. Do **not** merge leads/opportunities unless absolutely certain
+   they should be combined.
+
+When leads/opportunities should not be merged
+=============================================
+
+There may be instances where a similar lead or opportunity is identified, but should *not* be
+merged. These circumstances vary, based on the processes of the sales team and organization. Some
+potential scenarios are listed below.
+
+Lost leads
+----------
+
+If a lead/opportunity has been marked as :doc:`lost <lost_opportunities>`, it can still be merged
+with an active lead or opportunity. The resulting lead/opportunity is marked active, and added to
+the pipeline.
```

Comment:
Perhaps a quick concluding sentence explaining WHY this scenario shouldn't result in a merge could be beneficial to the reader.